[2023-09-24T07:18:08.226+0000] {param.py:354} DEBUG - Updating task params ({'example_key': 'example_value'}) with DagRun.conf ({'example_key': 'example_value'})
[2023-09-24T07:18:08.262+0000] {taskinstance.py:1092} DEBUG - previous_execution_date was called
[2023-09-24T07:18:08.448+0000] {__init__.py:51} DEBUG - Loading core task runner: StandardTaskRunner
[2023-09-24T07:18:08.498+0000] {param.py:354} DEBUG - Updating task params ({'example_key': 'example_value'}) with DagRun.conf ({'example_key': 'example_value'})
[2023-09-24T07:18:08.538+0000] {taskinstance.py:1092} DEBUG - previous_execution_date was called
[2023-09-24T07:18:08.586+0000] {base_task_runner.py:68} DEBUG - Planning to run as the  user
[2023-09-24T07:18:08.631+0000] {taskinstance.py:842} DEBUG - Refreshing TaskInstance <TaskInstance: example_bash_operator.runme_2 manual__2023-09-24T07:18:01+00:00 [queued]> from DB
[2023-09-24T07:18:08.735+0000] {taskinstance.py:1166} DEBUG - <TaskInstance: example_bash_operator.runme_2 manual__2023-09-24T07:18:01+00:00 [queued]> dependency 'Task Instance Not Running' PASSED: True, Task is not in running state.
[2023-09-24T07:18:08.736+0000] {taskinstance.py:1166} DEBUG - <TaskInstance: example_bash_operator.runme_2 manual__2023-09-24T07:18:01+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2023-09-24T07:18:08.736+0000] {taskinstance.py:1166} DEBUG - <TaskInstance: example_bash_operator.runme_2 manual__2023-09-24T07:18:01+00:00 [queued]> dependency 'Trigger Rule' PASSED: True, The task instance did not have any upstream tasks.
[2023-09-24T07:18:08.800+0000] {taskinstance.py:1166} DEBUG - <TaskInstance: example_bash_operator.runme_2 manual__2023-09-24T07:18:01+00:00 [queued]> dependency 'Task Instance State' PASSED: True, Task state queued was valid.
[2023-09-24T07:18:08.802+0000] {taskinstance.py:1166} DEBUG - <TaskInstance: example_bash_operator.runme_2 manual__2023-09-24T07:18:01+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2023-09-24T07:18:08.803+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_bash_operator.runme_2 manual__2023-09-24T07:18:01+00:00 [queued]>
[2023-09-24T07:18:08.832+0000] {taskinstance.py:1166} DEBUG - <TaskInstance: example_bash_operator.runme_2 manual__2023-09-24T07:18:01+00:00 [queued]> dependency 'Task Concurrency' PASSED: True, Task concurrency is not set.
[2023-09-24T07:18:08.833+0000] {taskinstance.py:1166} DEBUG - <TaskInstance: example_bash_operator.runme_2 manual__2023-09-24T07:18:01+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2023-09-24T07:18:08.900+0000] {taskinstance.py:1166} DEBUG - <TaskInstance: example_bash_operator.runme_2 manual__2023-09-24T07:18:01+00:00 [queued]> dependency 'Pool Slots Available' PASSED: True, There are enough open slots in default_pool to execute the task
[2023-09-24T07:18:08.901+0000] {taskinstance.py:1166} DEBUG - <TaskInstance: example_bash_operator.runme_2 manual__2023-09-24T07:18:01+00:00 [queued]> dependency 'Trigger Rule' PASSED: True, The task instance did not have any upstream tasks.
[2023-09-24T07:18:08.902+0000] {taskinstance.py:1166} DEBUG - <TaskInstance: example_bash_operator.runme_2 manual__2023-09-24T07:18:01+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2023-09-24T07:18:08.902+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_bash_operator.runme_2 manual__2023-09-24T07:18:01+00:00 [queued]>
[2023-09-24T07:18:08.902+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-09-24T07:18:08.983+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): runme_2> on 2023-09-24 07:18:01+00:00
[2023-09-24T07:18:09.034+0000] {standard_task_runner.py:57} INFO - Started process 817 to run task
[2023-09-24T07:18:09.050+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'runme_2', 'manual__2023-09-24T07:18:01+00:00', '--job-id', '56', '--raw', '--subdir', 'DAGS_FOLDER/example_bashoperator.py', '--cfg-path', '/tmp/tmp2w1rrzzf']
[2023-09-24T07:18:09.060+0000] {standard_task_runner.py:85} INFO - Job 56: Subtask runme_2
[2023-09-24T07:18:09.080+0000] {cli_action_loggers.py:67} DEBUG - Calling callbacks: [<function default_action_log at 0x7f199406d550>]
[2023-09-24T07:18:09.408+0000] {param.py:354} DEBUG - Updating task params ({'example_key': 'example_value'}) with DagRun.conf ({'example_key': 'example_value'})
[2023-09-24T07:18:09.489+0000] {taskinstance.py:1092} DEBUG - previous_execution_date was called
[2023-09-24T07:18:09.540+0000] {task_command.py:415} INFO - Running <TaskInstance: example_bash_operator.runme_2 manual__2023-09-24T07:18:01+00:00 [running]> on host 99e69330c0f0
[2023-09-24T07:18:09.541+0000] {settings.py:353} DEBUG - Disposing DB connection pool (PID 817)
[2023-09-24T07:18:09.542+0000] {settings.py:212} DEBUG - Setting up DB connection pool (PID 817)
[2023-09-24T07:18:09.543+0000] {settings.py:285} DEBUG - settings.prepare_engine_args(): Using NullPool
[2023-09-24T07:18:09.559+0000] {taskinstance.py:842} DEBUG - Refreshing TaskInstance <TaskInstance: example_bash_operator.runme_2 manual__2023-09-24T07:18:01+00:00 [running]> from DB
[2023-09-24T07:18:09.782+0000] {param.py:354} DEBUG - Updating task params ({'example_key': 'example_value'}) with DagRun.conf ({'example_key': 'example_value'})
[2023-09-24T07:18:09.866+0000] {taskinstance.py:1092} DEBUG - previous_execution_date was called
[2023-09-24T07:18:09.931+0000] {taskinstance.py:923} DEBUG - Clearing XCom data
[2023-09-24T07:18:09.986+0000] {retries.py:93} DEBUG - Running RenderedTaskInstanceFields.write with retries. Try 1 of 3
[2023-09-24T07:18:10.170+0000] {retries.py:93} DEBUG - Running RenderedTaskInstanceFields._do_delete_old_records with retries. Try 1 of 3
[2023-09-24T07:18:10.221+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_bash_operator' AIRFLOW_CTX_TASK_ID='runme_2' AIRFLOW_CTX_EXECUTION_DATE='2023-09-24T07:18:01+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-24T07:18:01+00:00'
[2023-09-24T07:18:10.222+0000] {__init__.py:116} DEBUG - Preparing lineage inlets and outlets
[2023-09-24T07:18:10.228+0000] {__init__.py:157} DEBUG - inlets: [], outlets: []
[2023-09-24T07:18:10.230+0000] {bash.py:186} DEBUG - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_bash_operator' AIRFLOW_CTX_TASK_ID='runme_2' AIRFLOW_CTX_EXECUTION_DATE='2023-09-24T07:18:01+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-24T07:18:01+00:00'
[2023-09-24T07:18:10.231+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-24T07:18:10.237+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "example_bash_operator__runme_2__20230924" && sleep 1']
[2023-09-24T07:18:10.315+0000] {subprocess.py:86} INFO - Output:
[2023-09-24T07:18:10.376+0000] {subprocess.py:93} INFO - example_bash_operator__runme_2__20230924
[2023-09-24T07:18:11.373+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-09-24T07:18:11.429+0000] {__init__.py:74} DEBUG - Lineage called with inlets: [], outlets: []
[2023-09-24T07:18:11.430+0000] {taskinstance.py:842} DEBUG - Refreshing TaskInstance <TaskInstance: example_bash_operator.runme_2 manual__2023-09-24T07:18:01+00:00 [running]> from DB
[2023-09-24T07:18:11.490+0000] {taskinstance.py:1456} DEBUG - Clearing next_method and next_kwargs.
[2023-09-24T07:18:11.491+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_2, execution_date=20230924T071801, start_date=20230924T071808, end_date=20230924T071811
[2023-09-24T07:18:11.492+0000] {taskinstance.py:2428} DEBUG - Task Duration set to 2.686907
[2023-09-24T07:18:11.558+0000] {cli_action_loggers.py:85} DEBUG - Calling callbacks: []
[2023-09-24T07:18:11.574+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-24T07:18:11.660+0000] {dagrun.py:734} DEBUG - number of tis tasks for <DagRun example_bash_operator @ 2023-09-24 07:18:01+00:00: manual__2023-09-24T07:18:01+00:00, state:running, queued_at: 2023-09-24 07:18:03.806398+00:00. externally triggered: True>: 7 task(s)
[2023-09-24T07:18:11.686+0000] {dagrun.py:755} DEBUG - number of scheduleable tasks for <DagRun example_bash_operator @ 2023-09-24 07:18:01+00:00: manual__2023-09-24T07:18:01+00:00, state:running, queued_at: 2023-09-24 07:18:03.806398+00:00. externally triggered: True>: 1 task(s)
[2023-09-24T07:18:11.695+0000] {taskinstance.py:1166} DEBUG - <TaskInstance: example_bash_operator.run_after_loop manual__2023-09-24T07:18:01+00:00 [None]> dependency 'Trigger Rule' PASSED: False, Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_states=_UpstreamTIStates(success=2, skipped=0, failed=0, upstream_failed=0, removed=0, done=2, success_setup=0, skipped_setup=0), upstream_task_ids={'runme_1', 'runme_0', 'runme_2'}
[2023-09-24T07:18:11.705+0000] {taskinstance.py:1147} DEBUG - Dependencies not met for <TaskInstance: example_bash_operator.run_after_loop manual__2023-09-24T07:18:01+00:00 [None]>, dependency 'Trigger Rule' FAILED: Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_states=_UpstreamTIStates(success=2, skipped=0, failed=0, upstream_failed=0, removed=0, done=2, success_setup=0, skipped_setup=0), upstream_task_ids={'runme_1', 'runme_0', 'runme_2'}
[2023-09-24T07:18:11.712+0000] {taskinstance.py:1166} DEBUG - <TaskInstance: example_bash_operator.run_after_loop manual__2023-09-24T07:18:01+00:00 [None]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2023-09-24T07:18:11.713+0000] {taskinstance.py:1166} DEBUG - <TaskInstance: example_bash_operator.run_after_loop manual__2023-09-24T07:18:01+00:00 [None]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2023-09-24T07:18:11.751+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
